play/wikirecent/app/utils/async_stream.py

```diff
@@ -0,0 +1,51 @@
+#!/usr/bin/env python3
+# Copyright Materialize, Inc. All rights reserved.
+#
+# Use of this software is governed by the Business Source License
+# included in the LICENSE file at the root of this repository.
+#
+# As of the Change Date specified in that file, in accordance with
+# the Business Source License, use of this software will be governed
+# by the Apache License, Version 2.0.
+
+"""async_stream
+
+Module showing an example for how to asynchronously stream a VIEW using websockets from the local
+Python web server.
+"""
```

Comment:
this is intended as just a much simpler thing than the js page, right?

Reply:
Yeah, it's meant to be both simpler, but also an isolated example. I'll document this better.